### PR TITLE
Do not raise error if `PRIMER_WARNINGS_DISABLED` is set.

### PR DESCRIPTION
**Pull request by joelhawksley**
We've seen some friction from this error message, so I've added the option to disable it via an ENV flag.

cc @phillmv

### CODE REVIEW

File: app/lib/primer/classify.rb [hunk 1]
```diff
@@ -123,13 +123,13 @@ def call(classes: "", style: nil, **args)
       def validated_class_names(classes)
         return if classes.blank?
 
-        if ENV["RAILS_ENV"] == "development"
+        if ENV["RAILS_ENV"] == "development" && !ENV["PRIMER_WARNINGS_DISABLED"]
```
Comment (manuelpuyol) [inline app/lib/primer/classify.rb:126]: could we use an engine config instead? Something similar to `silence_deprecations`

File: test/lib/classify_test.rb [hunk 2]
```diff
@@ -626,6 +626,16 @@ def test_raises_error_when_passing_in_a_primer_css_class_name_in_development
     ENV["RAILS_ENV"] = "test"
   end
 
+  def test_raises_does_not_raise_error_when_passing_in_a_primer_css_class_name_in_development_and_flag_is_set
+    ENV["RAILS_ENV"] = "development"
+    ENV["PRIMER_WARNINGS_DISABLED"] = "1"
+
+    assert_generated_class("bg-blue text-center float-left ml-1", { classes: "bg-blue text-center float-left ml-1" })
+  ensure
+    ENV["PRIMER_WARNINGS_DISABLED"] = "0"
```
Comment (manuelpuyol) [inline test/lib/classify_test.rb:635]: we'd need to unset this var, or set it to nil to actually reset
`!ENV["PRIMER_WARNINGS_DISABLED"] == !"0" == false`

File: app/lib/primer/classify.rb (hunk 1 above)
Comment (joelhawksley) [reply app/lib/primer/classify.rb:126]: 🤔 we could, but I was hoping to make this an easy command line flag, IE: `PRIMER_WARNINGS_DISABLED=1 script/server`

Comment (phillmv) [reply app/lib/primer/classify.rb:126]: FWIW the use case here is:

"I have to fix a bug in dotcom, but I'm getting a deprecation error when looking at the page. I don't own the view, or the fifty places it's linked, so I guess I'll just go and comment out the error raise in `/vendor`" 😉.

**Pull request merged**